sql-plugin/src/main/scala/com/nvidia/spark/rapids/aggregate.scala

```diff
@@ -37,6 +37,40 @@ import org.apache.spark.sql.rapids.{CudfAggregate, GpuAggregateExpression, GpuDe
 import org.apache.spark.sql.types.{DoubleType, FloatType}
 import org.apache.spark.sql.vectorized.{ColumnarBatch, ColumnVector}
 
+object AggregateUtils {
+
+  private val aggs = List("min", "max", "avg", "sum", "count", "first", "last")
```

Comment:
Kuhu suggested it would be good if we can get above values from AggregateFunctions.scala. I am trying to figure out how to retrive these values.

Reply:
This is a nit and if it is not straight forward, please keep the way it is.